jetty9/src/main/java/com/linecorp/armeria/server/jetty/JettyService.java

```diff
@@ -318,6 +318,9 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) {
                 });
                 success = true;
                 return null;
+            } catch (Exception e) {
+                res.close(e);
```

Comment:
I prefer `.abort(e)` because `.close(e)` does not clean up internal resources.
A byte array is used to convey the data though.

Reply:
Done!